src/terraform-resource/in/in.go

```diff
@@ -65,6 +65,15 @@ func (r Runner) Run(req models.InRequest) (models.InResponse, error) {
 		StateFileLocalPath:  path.Join(tmpDir, "terraform.tfstate"),
 		StateFileRemotePath: stateFilename,
 	}
+
+	if req.Params.OutputModule != "" {
+		terraformModel = models.Terraform{
```

Comment:
Could remove a bit of duplication by saying `terraformModel.OutputModule = ...` rather than re-declaring the whole struct.